### ISSUE TITLE
`VkCuFunctionCreateInfoNVX::pName` char-array should have a length (`null-terminated`?)

### ISSUE THREAD

**Issue by MarijnS95** [KhronosGroup/Vulkan-Docs#1526]
The `pName` string member of `VkCuFunctionCreateInfoNVX` lacks a `len="null-terminated"` annotation, leading to our binding generator failing to understand how long the pointed-to array is supposed to be. I believe this should be treated like any other string (`char*`) in `vk.xml`, null-terminated instead of fixed length?

https://github.com/KhronosGroup/Vulkan-Docs/blob/dfe3bcd0e1b7239e2d6ae8b6afe563780a711854/xml/vk.xml#5820:

```xml
        <type category="struct" name="VkCuFunctionCreateInfoNVX">
            <member values="VK_STRUCTURE_TYPE_CU_FUNCTION_CREATE_INFO_NVX"><type>VkStructureType</type> <name>sType</name></member>
            <member>const <type>void</type>* <name>pNext</name></member>
            <member><type>VkCuModuleNVX</type>          <name>module</name></member>
            <member>const <type>char</type>*            <name>pName</name></member>
        </type>
```

**Comment by oddhack** [comment]
@dgkoch this really should be assigned to Eric Werness or Liam Middlebrook but they don't seem to be in the KhronosGroup group here so I can't. This is a very weird extension with XML but no spec text yet, so there's no way to refer to the text for what this member should be tagged as.

**Comment by liam-middlebrook** [comment]
That's odd. I'm a member of the KhronosGroup org here (although my GitHub username does differ from my Khronos username, I use my GitHub username on extension contact fields).

Thanks for reporting this Marijn, I'll try and get this resolved before the next specification release.

**Comment by oddhack** [comment]
> That's odd. I'm a member of the KhronosGroup org here (although my GitHub username does differ from my Khronos username, I use my GitHub username on extension contact fields).
> 
> Thanks for reporting this Marijn, I'll try and get this resolved before the next specification release.

Thanks, Liam. I may have mistyped something in the assignee search box and do see you now.

**Comment by MarijnS95** [comment]
Thanks @liam-middlebrook, appreciated!

Would you also be able to look into `VK_VIDEO_DECODE_H264_PROGRESSIVE_PICTURES_ONLY_EXT` which should be called `VK_VIDEO_DECODE_H264_FIELD_LAYOUT_PROGRESSIVE_PICTURES_ONLY_EXT` (notice missing `_FIELD_LAYOUT` in the current name)?

Likewise, `VK_VIDEO_ENCODE_H264_CAPABILITY_*` variants should use plural `CAPABILITIES_` in the name so that it has a common prefix with its enum type name `VkVideoEncodeH264CapabilitiesFlagBitsEXT` again.

**Comment by dgkoch** [comment]
The video issues are likely better reported in a separate issue (different folks are responsible for those).

**Comment by MarijnS95** [comment]
@dgkoch Oh disregard this entirely - I somehow put this in the wrong place :)

**Comment by oddhack** [comment]
This should be fixed in the 1.2.179 spec update.
